src/server/index.jsx

```diff
@@ -205,7 +205,7 @@ server.get(
         isLite: isLiteRouteSuffix,
         route: { getInitialData, pageType },
         variant,
-      } = getRouteProps(urlPath);
+      } = getRouteProps(url);
```

Comment:
Not sure if this is really needed. This is just to maintain query params when using the script switch button, as this value eventually sets the RequestContext `pathname` value.

Reply:
This actually mirrors Next.js though. We use `context.resolvedUrl` there which includes the entire URL structure and pass that to `RequestContext.pathname`

Reply:
Changed this, so no longer applicable